api/web.config

```diff
@@ -0,0 +1,88 @@
+        </rule>
+        <rule name="version/1.0" stopProcessing="true">
+          <match url="^version/(1.0|2.0)$" />
+          <action type="Rewrite" url="versioninfo.php?version={R:1}" appendQueryString="true" />
+        </rule>
+        <rule name="version/platform" stopProcessing="true">
+          <match url="^version/(android|ios|linux|mac|web|windows|developer)$" />
+          <action type="Rewrite" url="versioninfo.php?platform={R:1}" appendQueryString="true" />
+        </rule>
+        <rule name="version/platform/v" stopProcessing="true">
+          <match url="^version/(android|ios|linux|mac|web|windows|developer)/(1.0|2.0)$" />
+          <action type="Rewrite" url="versioninfo.php?platform={R:1}&amp;version={R:2}" appendQueryString="true" />
+        </rule>
+        <!-- history -->
+        <rule name="history/platform" stopProcessing="true">
+          <match url="^history/(android|ios|linux|mac|web|windows|developer)$" />
+          <action type="Rewrite" url="history.php?platform={R:1}" appendQueryString="true" />
+        </rule>
+        <!-- keyboard -->
+        <rule name="keyboard/v" stopProcessing="true">
+          <match url="^keyboard/(1.0)/(.+)$" />
+          <action type="Rewrite" url="keyboard.php?version={R:1}&amp;id={R:2}" appendQueryString="true" />
+        </rule>
+        <rule name="keyboard/" stopProcessing="true">
+          <match url="^keyboard/(.+)$" />
+          <action type="Rewrite" url="keyboard.php?id={R:1}" appendQueryString="true" />
+        </rule>
+        <!-- model -->
+        <rule name="model/v" stopProcessing="true">
+          <match url="^model/(1.0)/(.+)$" />
+          <action type="Rewrite" url="model.php?version={R:1}&amp;id={R:2}" appendQueryString="true" />
+        </rule>
+        <rule name="model/" stopProcessing="true">
+          <match url="^model/(.+)$" />
+          <action type="Rewrite" url="model.php?id={R:1}" appendQueryString="true" />
+        </rule>
+        <!-- bundles -->
+        <rule name="bundles/" stopProcessing="true">
+          <match url="^bundles(/?)$" />
+          <action type="Rewrite" url="bundles.php?q=all" appendQueryString="true" />
+        </rule>
+        <rule name="bundles/*" stopProcessing="true">
+          <match url="^bundles/(.+)$" />
+          <action type="Rewrite" url="bundles.php?q={R:1}" appendQueryString="true" />
+        </rule>
+        <rule name="bundles/v" stopProcessing="true">
+          <match url="^bundles/1.0(/?)$" />
+          <action type="Rewrite" url="bundles.php?q=all" appendQueryString="true" />
+        </rule>
+        <rule name="bundles/v/*" stopProcessing="true">
+          <match url="^bundles/1.0/(.+)$" />
+          <action type="Rewrite" url="bundles.php?q={R:1}" appendQueryString="true" />
+        </rule>
+      </rules>
+    </rewrite>
+  </system.webServer>
+</configuration>
+
+<!--
+RewriteEngine On
+RewriteRule ^version$ versioninfo.php [L]
```

Comment:
Any reason we're keeping this commented-out section?

Reply:
Not really any reason -- I guess I was using it as a comparison strategy with .htaccess but it's probably going to be unhelpful in the longer term anyway.